README.md

```diff
@@ -135,6 +135,25 @@ If your gpu count per node is not 8, adjust:
 
 in the SBATCH command section.
 
+## Enable Float8 Training on H100s
+
+Install latest [TorchAO](https://github.com/pytorch/ao/tree/main/torchao/float8) so we can import float8 dtype
+```
+USE_CPP=0 python -m pip install git+https://github.com/pytorch/ao.git
+```
+
+Launch training job with following command
+```
+CONFIG_FILE="./train_configs/llama3_8b.toml" ./run_llama_train.sh --float8.enable_float8_linear --float8.enable_fsdp_float8_all_gather --float8.precompute_float8_dynamic_scale_for_fsdp
+```
+* `--float8.enable_float8_linear`: swap `nn.Linear` with `Float8Linear` to perform float8 matmul.
+* `--float8.enable_fsdp_float8_all_gather`: cast `Float8Linear.weight` from high precision to float8 before FSDP all-gather so we can communicate in float8 to save bandwidth.
+* `--float8.precompute_float8_dynamic_scale_for_fsdp` [optional]: communicate AMAX/scales efficiently in a single all-reduce for all parameters instead of doing many small all-reduce for each parameters.
+
+For scaling strategy, we are supporting tensor-wise scaling with dynamic scales. We are actively working on tensor-wise saling with delayed scales. We are also exploring row-wise sclaing
+
+For parallelsims, we are supporting FSDP float8 all-gather. We are actively working on float8 all-gather in TP.
```

Comment:
I thought fp8 allgather in TP already enabled? What are the gaps currently?

Reply:
Yes, TP float8 all-gather is enabled in H1. Last week we had async TP + float8 all-gather working so we are refreshing 2D results. I want to announce TP availablility in the next dev-discuss post. What do you think?

Reply:
I corrected my word and now we say float8 + TP are supported

Reply:
My thoughts on this: 

I think TP + fp8 itself can be a separate post yes!

I think specifically async overlap when applying TP + fp8 can be a section on that post. Basically we still need some data points show to the user that whether turning on async overlap when using fp8 allgather in TP is beneficial, given that fp8 TP allgather already halve the allgather latency, the performance improvement of async overlap during real training might not be significant compare to bf16

Reply:
True. Yifu benchmarked async TP + fp8 and the speed up is less significant than async TP brings to bf16, but there is some speed up. Will share updated 2D results soon